libraries/Resample/aa_fir.h

```diff
@@ -0,0 +1,2935 @@
+#include <vector>
```

Comment:
Not sure how I feel about dumping such a large file in here ... this file limits the resampling options that are readily accessible to the user to the ones that you selected at the design stage. Maybe a better option is to have a way of calling the python script from C++ to generate a filter table on the fly? This could be achieved via `popen()` call if there are no better options (e.g.: some standardised way of calling python from C). Then one could have instructions for the user to generate their own header files if they wanted to avoid the runtime overhead.

Reply:
It's true that this way the decimation and interpolation factors are fixed to 2, 4, 8 and 16 and one could use [many more](https://www.wolframalpha.com/input/?i=devisors+of+44100). I like your proposal, but the script is dependent on scipy. It doesn't seem available on the Bela. Any ideas?